src/ecs/components/Model.zig

```diff
@@ -0,0 +1,41 @@
+const std = @import("std");
+
+const main = @import("main");
+
+const graphics = main.graphics;
+
+const Image = graphics.Image;
+
+const ZonElement = main.ZonElement;
+
+const Model = @This();
+
+texture: main.graphics.Texture = undefined,
```

Comment:
Ideally we should start with a server-side only ECS for now, since the client doesn't (and shouldn't) have most of the data, additionally the client side does need to update fewer things.
So an ECS is probably not really needed on the client and might even end up hurting performance.